include/utils.php

```diff
@@ -109,6 +109,26 @@ function make_sugar_config(&$sugar_config)
 
     $sugar_config = array(
         'admin_export_only' => empty($admin_export_only) ? false : $admin_export_only,
+        'anti_malware_scanners' => array(
+            'SuiteCRM\Utility\AntiMalware\Providers\ClamTCP' => array(
+                'name' => 'ClamAV TCP',
+                'support_page' => 'https://www.clamav.net/',
+                'enabled' => false,
+                'path' => null,
+                'options' => array(
+                    'ip' => '127.0.0.1',
+                    'port' => 3310,
+                    'type' => 'local'
+                ),
+            ),
+            'SuiteCRM\Utility\AntiMalware\Providers\Sophos' => array(
+                'name' => 'Sophos Anti Virus (Linux)',
+                'support_page' => 'https://www.sophos.com/en-us/products/free-tools/sophos-antivirus-for-linux.aspx',
+                'enabled' => false,
+                'path' => '/opt/sophos-av/bin/savscan',
+                'options' => '-ss'
+            )
+        ),
```

Comment:
I believe this can be removed. From what I can see this ```$sugar_config``` array is intended to populate values during setup, as we aren't actually populating anything here we only need to use the ```$sugar_config_defaults``` below.